packages/integration/lib/src/views/compose/compose_screen.dart

```diff
@@ -38,8 +38,14 @@ class LMFeedComposeScreen extends StatefulWidget {
   final LMFeedComposeScreenStyle? style;
 
   final Function(BuildContext context)? composeDiscardDialogBuilder;
-  final PreferredSizeWidget Function(LMFeedAppBar oldAppBar)?
-      composeAppBarBuilder;
+  final PreferredSizeWidget Function(
```

Comment:
add a documentation comment here for the `composeAppBarBuilder` to explain all the parameters.